api/api.go

```diff
@@ -266,6 +266,9 @@ func (api *API) Exec(request *frames.ExecRequest) (frames.Frame, error) {
 }
 
 func (api *API) History(request *frames.HistoryRequest, out chan frames.Frame) error {
+	if api.historyServer == nil {
+		return errors.New("history server was not initialized properly. restart framesd to access this feature")
```

Comment:
Can we save the reason it failed to initialize and report it here? Otherwise, this is super annoying for the user (because init will obviously fail again if they restart the service).

Reply:
Also to nitpick: `restart` -> `Restart`, `access` -> `enable`

Reply:
i log it in the frames pod log. the error will probably say nothing to the user

Reply:
It's much easier to get the error here than have to go search for it in the beginning of the pod log...

Reply:
We decided to tell the user to contact their system administrator.